src/ansys/fluent/core/services/app_utilities.py

```diff
@@ -0,0 +1,247 @@
+"""Wrappers over AppUtilities gRPC service of Fluent."""
+
+from enum import Enum
+from typing import List, Tuple
+
+from google.protobuf.json_format import MessageToDict
+import grpc
+
+from ansys.api.fluent.v0 import app_utilities_pb2 as AppUtilitiesProtoModule
+from ansys.api.fluent.v0 import app_utilities_pb2_grpc as AppUtilitiesGrpcModule
+import ansys.fluent.core as pyfluent
```

Comment:
@hpohekar Please focus your imports to the things thath your code needs. We should not see this anywhere inside the code.